Seed/Characters/Player.cs

```diff
@@ -476,7 +481,35 @@ public void Watch(Location location)
 
         public void Watch(Character character)
         {
+            Console.ForegroundColor = ConsoleColor.White;
             Service.DisplayLongString("\n" + character.Overview + "\n");
+
+            Console.Write($"{character.Name} jest ");
+            if(character.HP==character.MaxHP)
```

Comment:
could be refactored to a method isCharacterFullHP, to contain the logic within the character class